scripts/generate-notices.py

```
@@ -0,0 +1,235 @@
+def now_utc() -> str:
+    return datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
+
+
+def operator_notices():
+    gl = OPERATOR_DIR / "bin" / "go-licenses"
+    gl_path = str(gl) if gl.exists() else (shutil.which("go-licenses") or "")
+    if not gl_path:
+        sys.exit("ERROR: go-licenses not found. Run 'make -C operator go-licenses'.")
+    env = {**os.environ, "GOFLAGS": "-mod=vendor"}
+    stdlib = subprocess.check_output(["go", "list", "std"], cwd=OPERATOR_DIR, env=env, text=True)
+    stdlib_ignore = ",".join(sorted({l.split("/")[0] for l in stdlib.splitlines() if l}))
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)